src/car.cpp

```diff
@@ -154,7 +173,68 @@ void Car::move(vec2 off)
 	m_position.y += off.y;
 }
 
+void Car::set_lane(int n)
+{
+	m_lane = n;
+}
+
+int Car::get_lane()
+{
+	return m_lane;
+}
+
 void Car::set_rotation(float radians)
 {
 	m_rotation = radians;
 }
+
+void Car::slow_down()
+{
+	m_velocity.x = m_max_speed-m_acceleration.x; // gets the update loop running again, probably change to a smarter way within the update conditional
+	m_acceleration.x *= -1.f;
+	m_acceleration.y = 0.f;

```

Comment:
can you maybe add a TODO to expand this function to also include the y component of the velocity/acceleration? like we don't need to do it for this PR, but we'll need to do it soon, so it's probably worthwhile to make a note of where we'll need to make a change